samples/excel/tables/add-rows-to-a-table.yaml

```diff
@@ -0,0 +1,164 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Add rows to a table
+description: Add rows to a table
+script:
+  content: |+
+    $('#create-table').click(createTable);
+    $('#add-rows').click(addRows);
+
+    /** Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // If sheet already exists, queue a command to delete it
+                context.workbook.worksheets.getItem("Playground").delete();
+
+                // Then create a proxy object for a new worksheet
+                sheet = context.workbook.worksheets.add("Playground");
+
+                // Call the helper to create the table
+                await queueCreateTable(sheet);
+
+                //Queue a command to activate the sheet
+                sheet.activate();
+
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                // Try to activate the new sheet regardless, to show
+                // how far the processing got before failing
+                sheet.activate();
+                await context.sync();
+
+                // Then re-throw the original error, for appropriate error-handling
+                // (in this snippet, simply showing a notification)
+                throw error;
+            }
+        })
+        .catch(handleError);
+    }
+
+
+    /** Add rows to an existing table */
+    function addRows() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to get the sheet
+                sheet = context.workbook.worksheets.getItem("Playground");
+
+                // Call the helper to sort the table
+                await queueAddRows(sheet);
+
+                // Run all of the queued up commands
+                await context.sync();
```

Comment:
Not sure the comment before .sync() is needed. We should discuss whether this should be added everywhere.